app/src/renderer/connectors/lcdClient.js

```diff
@@ -204,8 +204,58 @@ const Client = (axios, remoteLcdURL) => {
         [].concat(depositorTxs, proposerTxs)
       )
     },
+    /* ============ Explorer ============ */
     getBlock: function(blockHeight) {
       return req(`GET`, `/blocks/${blockHeight}`)()
+    },
+    /* ============ Distribution ============ */
+    getDelegatorRewards: function(delegatorAddr) {
+      return req(`GET`, `/distribution/delegators/${delegatorAddr}/rewards`)()
+    },
+    postDelegatorRewards: function(delegatorAddr, data) {
```

Comment:
what does this do?

Reply:
withdraws all rewards from a delegator

Reply:
from all the delegations at once

Reply:
how about:
```suggestion
    postDelegationRewardsClaim: function(delegatorAddr, data) {
```

Reply:
or:
```suggestion
    postDelegatorRewardsClaim: function(delegatorAddr, data) {
```

Reply:
what about `postWithdrawDelegatorRewards` ?

Reply:
good for me

Reply:
updated